docs/items/overrides.md

```diff
@@ -12,21 +12,25 @@ To add a property to an `Item`, simply call `addPropertyOverride` on it. The `Re
 Using Overrides
 ---------------
 
-The format of an override can be seen on the [wiki][], and a good example can be found in `model/item/bow.json`. For reference, here is a hypothetical example of an item with a `power` property. Notice that when you define a predicate, it applies to *all values __greater than or equal to__ the given value*. If the values have no match, the default is the current model.
+The format of an override can be seen on the [wiki], and a good example can be found in `model/item/bow.json`. For reference, here is a hypothetical example of an item with a `power` property. Notice that when you define a predicate, it applies to *all values __greater than or equal to__ the given value*. If the values have no match, the default is the current model.
 
+```json
+{
+  "parent": "item/generated",
+  "texture": {
+    "__comment": "Default",
+    "layer0": "examplemod:items/examplePartial"
+  },
+  "overrides": [
     {
-      "parent": "item/generated",
-      "texture": {
-        "layer0": "examplemod:items/examplePartial" // Default
+      "__comment": "Power >= 0.75",
```

Comment:
Are they? Definitely not in the official JSON specification. And Gson only supports them in lenient mode, which is not enabled at least in forge's blockstate loader.